apk-generator/v2/app/generator/generator.py

```diff
@@ -42,7 +42,8 @@ def __init__(self, config, via_api=False, identifier=None, task_handle=None):
         self.app_launcher_icon = os.path.abspath(config['BASE_DIR'] + '/app/static/assets/ic_launcher.png')
         self.app_package_name = 'org.fossasia.openevent.' + self.app_name.replace(" ", "")
         self.app_temp_assets = os.path.abspath(self.working_dir + '/' + self.identifier + '/assets-src/')
-        self.api_link = ''
+        self.app_temp_images_speakers = os.path.abspath(self.working_dir + '/' + self.identifier + '/drawable-res-src/')
+	self.api_link = ''
```

Comment:
@arp95 You have mixed tabs and spaces here. Please fix it and confirm by running the generator